src/scripts/generate-test-patient-spreadsheets.sh

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env bash
+set -euo pipefail
+if [ "${DEBUG:=false}" == "true" ]; then set -x; fi
```

Comment:
This script uses secrets as vars... I think debug probably shouldn't be allowed through the environment.